tuning-fork/boilerplate
# Fix login popups appearing when entering incorrect creds

## 🤺 Summary

Closes #392 

## 📝 Checklist


## 🔬 Steps to test
1. Go to splashpage
2. Enter invalid creds
3. Don't expect to have any popups, just the error message should appear.
4. Enter valid creds
5. Expect popup saying you logged in, and then to be redirected to the dashboard
